src/locale/en_US/LC_MESSAGES/django.po

```diff
@@ -4305,7 +4305,7 @@ msgstr "biography"
 msgid ""
 "Describe yourself with 500 characters or less. There will be no formatting."
```

Comment:
Though your changes make the users see what we want them to see, the message id still shows the old information. Please also change:

- the help text in `src/users/models.py`
- the message id in the `src/locale/{en_US, zh_Hant}/LC_MESSAGES/django.po`